app/lib/frontend/template_consts.dart

```diff
@@ -125,13 +125,13 @@ String _landingPageTitle(String platform) {
 final Map<String, String> _landingBlurbs = const {
   'default':
       '<p class="text">Find and use packages to build <a href="/flutter">Flutter</a> and '
-      '<a href="/web">web</a> apps with <a target="_blank" href="https://www.dartlang.org">Dart</a>.</p>',
+      '<a href="/web">web</a> apps with <a target="_blank" rel="noopener" href="https://www.dartlang.org">Dart</a>.</p>',
   KnownPlatforms.flutter:
-      '<p class="text"><a href="https://flutter.io/">Flutter<sup><small>↗</small></sup></a> '
+      '<p class="text"><a target="_blank" rel="noopener" href="https://flutter.io/">Flutter<sup><small>↗</small></sup></a> '
```

Comment:
This wasn't using target `_blank` before, but I assume that change was deliberate too.

Reply:
Yes, deliberate change, I think this way it is more consistent with the other links.